pyproject.toml

```diff
@@ -191,7 +191,7 @@ exclude_lines = [
   'except ImportError as e:',
   'except ImportError:',
```

Comment:
```suggestion
  'except ImportError.*',
```

Reply:
Also, consider using https://github.com/asottile/covdefaults
It also has conditional coverage.

Reply:
> Also, consider using https://github.com/asottile/covdefaults It also has conditional coverage.

Thanks for the suggestion. I'll experiment with adopting it in a follow up.